[KOGITO-2594] Implement script to generate Kogito example images (#414)
kiegroup/kogito-cloud-operator: README.md

- `container_engine` engine used to interact with images and local containers.
  *Default is docker.*
- `domain_suffix` domain suffix used for exposed services. Ignored when running tests on Openshift.

> - `image_cache_mode` Use this option to specify whether you want to use image cache for runtime images. Available options are always, never of if-available(default).

Reviewer: ```suggestion
- `image_cache_mode` Use this option to specify whether you want to use image cache for runtime images. Available options are 'always', 'never' or 'if-available'(default).
```
